administrator/language/en-GB/plg_fields_media.ini

```
@@ -3,9 +3,13 @@
 ; License GNU General Public License version 2 or later; see LICENSE.txt
 ; Note : All ini files need to be saved as UTF-8
 
+PLG_FIELDS_ACCESSIBLEMEDIA_LABEL="Media"
+PLG_FIELDS_ACCESSIBLEMEDIA_PARAMS_IMAGEFILE_LABEL="Image"
+PLG_FIELDS_ACCESSIBLEMEDIA_PARAMS_ALT_TEXT_LABEL="Alternative Text for Image"	
```

```suggestion
PLG_FIELDS_ACCESSIBLEMEDIA_PARAMS_ALT_TEXT_LABEL="Alt Text"	
```

To  be consistent with the text under `Images and Links` tab.